src/core.c

```diff
@@ -460,6 +460,9 @@ typedef struct CoreData {
             pthread_t threadId;             // Gamepad reading thread id
             int streamId[MAX_GAMEPADS];     // Gamepad device file descriptor
             char name[64];                  // Gamepad name holder
```

Comment:
Instead of creating an additional variable, this one can be used just adding the array:
```c
char name[MAX_GAMEPADS][64];
```

Reply:
Yes ok I updated it to use the existing variable which is now an array. It exposes these extra properties to the PLATFORM_WEB version is this ok?

![image](https://user-images.githubusercontent.com/10982154/132984626-69e441c7-d0e6-4ddc-a994-cf8de1b54a7a.png)

Reply:
yeah, I think it's ok for now